test/assets/preset-mini-targets.ts

```diff
@@ -848,13 +848,18 @@ export const presetMiniTargets: string[] = [
   'transition-colors,opacity-200',
   'transition-color,background-color-200',
   'transition-background-color,color-200',
+  'transition-$variant',
+  'transition-[width_height]',
+  'transition-[width_height_colors]',
+  'transition-[width_height_colors]-200',
```

Comment:
`transition-[width_height_colors]-200` This is a weird way of writing, I don't like it.